packages/editor/src/components/rich-text/index.js

```diff
@@ -215,17 +212,29 @@ export class RichText extends Component {
 	 * @param {PasteEvent} event The paste event as triggered by TinyMCE.
 	 */
 	onPaste( event ) {
-		const dataTransfer =
-			event.clipboardData ||
-			event.dataTransfer ||
-			this.editor.getDoc().dataTransfer ||
-			// Removes the need for further `dataTransfer` checks.
-			{ getData: () => '' };
-
-		const { items = [], files = [], types = [] } = dataTransfer;
+		const clipboardData = event.clipboardData;
+		const { items = [], files = [] } = clipboardData;
 		const item = find( [ ...items, ...files ], ( { type } ) => /^image\/(?:jpe?g|png|gif)$/.test( type ) );
-		const plainText = dataTransfer.getData( 'text/plain' );
-		const HTML = dataTransfer.getData( 'text/html' );
+		let plainText = '';
+		let HTML = '';
```

Comment:
I know this was all capitalised like this beforehand, but would calling this `html` instead of `HTML` make more sense? First case being uppercase implies a component/constant by convention to me. 🤷‍♂️

Reply:
Sure!

Reply:
We already have `HTML` as an argument in the raw handler though, so we'd have to do `HTML: html`.

Reply:
Okay, adjusted :)